test/e2e/predictor/test_tensorflow.py

```diff
@@ -50,3 +50,6 @@ def test_tensorflow_kfserving():
 
     KFServing.create(isvc)
     KFServing.wait_isvc_ready(service_name, namespace=KFSERVING_TEST_NAMESPACE)
+
+    # Delete the InferenceService
```

Comment:
looks this test is missing calling the prediction endpoint, would be mind adding that?

Reply:
Sure. Will do.

Reply:
Can we automate this into a library?

Create, Predict, Delete?

Reply:
Sorry for my late response since I was in doc-sprint last week.

@yuzisun I have added the call to prediction endpoint. Please review and let me know if further change is needed.

@ellis-bigelow 
I assume you mean to create a function to call "create, predict, delete"?  If yes, we need to pass in the following information, which seems a lot to me. What do you think?
- service_name
- spec type (to decide V1alpha2TensorflowSpec, V1alpha2PyTorchSpec, etc.)
- storage_uri
- requests cpu, requests memory  <-- e.g. tensorflow flower sample need more resources
- limits cpu, limits memory
- annotations
- path to input data (for predict)
- assert statement  <-- the function can probably return "predictions" and let calling function to call assert()